src/vmime/constants.cpp

```diff
@@ -232,5 +233,19 @@ namespace dispositionModifiers {
 	const char* const ERROR = "error";
 }
 
+// Constants for DSN (delivery status notification)
+namespace dsn
```

Comment:
Code style (brace at the end: "namespace dsn {").

Reply:
Sorry, I did not realize the code style difference, fixed.